### PR TITLE
9.0 [MIG] marketing_crm_partner migration

### PR DESCRIPTION
**Pull request by cubells**
- Imported last updates from v8
- Adapted to v9
- Updated README

@Tecnativa

### CODE REVIEW

**Comment by rafaelbn** [comment]
Thanks @cubells ! could you please review travis?

**Review by susport — APPROVED** [review]
Good 👍

File: marketing_crm_partner/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# © 2016 Tecnativa S.L. - Jairo Llopis
+# © 2016 Tecnativa S.L. - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Tracking Fields in Partners",
+    "summary": "Copy tracking fields from leads to partners",
+    "version": "9.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "website": "http://www.antiun.com",
+    "author": "Antiun Ingeniería S.L., Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline marketing_crm_partner/__openerp__.py]: Switch `Antiun` by `Tecnativa` here and change website too.

**Review by pedrobaeza — APPROVED** [review]
Little comment, but :+1:

**Pull request merged**